spec/jobs/audit_job_spec.rb

```diff
@@ -12,24 +12,23 @@
 
   let(:job) { AuditJob.new(file.id, 'content', uri) }
 
-  describe "audit on content" do
-    let(:uri) { file.content.uri }
-    it "should pass" do
-      expect(job.run).to eq(true)
+  describe "#run" do
+    subject { job.run }
+    context "on content" do
+      let(:uri) { file.content.uri }
+      it { is_expected.to be true }
     end
-  end
 
-  describe "audit on a version of the content" do
-    let(:uri) { file.content.versions.first.uri }
-    it "should pass" do
-      expect(job.run).to eq(true)
+    context "on a version of the content" do
+      before { Sufia::VersioningService.create(file.content) }
+      let(:uri) { Sufia::VersioningService.latest_version_of(file.content).uri }
+      it { is_expected.to be true }
     end
-  end
 
-  describe "audit on an invalid version of the content" do
-    let(:uri) { file.content.versions.first.uri + "bogus" }
-    it "should fail" do
-      expect(job.run).to eq(false)
+    context "on an invalid version of the content" do
+      before { Sufia::VersioningService.create(file.content) }
+      let(:uri) { Sufia::VersioningService.latest_version_of(file.content).uri + 'bogus' }
```

Comment:
I'm just trying to keep similar to the spirit of the test that was already there.